### DISCUSSION TITLE
Databricks integration would be great

### DISCUSSION THREAD

**Discussion by nandrzej** [grai-io/grai-core#665] [Feature Requests]
Data Lakehouse Platform is a popular cloud data warehouse competing with Snowflake. It would be great if Grai supported it.

**Comment by ieaves** [comment]
Hey @nandrzej, this is awesome. I don't have a ton of personal experience with the databricks universe outside of Spark and MLflow. We are currently working on an openlineage integration which will provide support for Spark but were there any specific component (like Delta Lake) you think would be most valuable?

**Comment by jordandakota** [comment]
There's also the new unity catalog open source implementation, would be good to both integrate with Databricks Unity and Unity OSS. An integration with Databricks would mostly be useful to connect to Unity and get all of the metadata directly out of there.
